system/HamburgerMenu.module.scss

```diff
@@ -0,0 +1,97 @@
+.body {
+  padding: 40px;
+  color: inherit;
+}
+
+.hamburgerContainer {
+  display: flex;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+  z-index: 10;
+
+  &.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  &.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  &.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background: inherit;
+  background-color: currentColor;
+  color: inherit;
+}
+
+.activeNavItemSquiggle {
+  height: 100%;
+  position: relative;
+  top: 0;
+  width: 0;
+  transition: width 0.5s ease-in-out;
+
+  overflow: visible;
+  width: auto;
+}
+
+.navbar {
+  position: fixed;
+  // (xBalbinus) In production, you would probably change this to 0 to have it at the top of the page
+  // This 49px is a hacky accounting for the Navigation component's height
+  top: 49px;
+  left: -100%;
+  width: 20%;
```

Comment:
I don't like this percent width it feels pretty jank.

Reply:
Fixed to use the min-width strategy, same as the nav.